### PR TITLE
Adjusting the wording as requested

### LINKED ISSUE #922: Wrong wording
- Wrong text displayed after submitting step 2 of `liquidity`  

- Should say `Please carefully read the instructions below` instead of `Please carefully read the instructions above`. 

Text is in [this](https://github.com/gnosis/dex-react/blob/fa2952418334ea36c694d463b591df07ee179148/src/components/PoolingWidget/AddFunding.tsx) file.

Image: 

Screenshot 2020-04-15 at 21 07 56

### PR DESCRIPTION
**Pull request by alfetopito**
Closes #922

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr923--dexreact.review.gnosisdev.com

File: src/components/PoolingWidget/AddFunding.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const AddFunding: React.FC<AddFundingProps> = ({ txIdentifier, txReceipt }) => (
         </strong>
         <p>
           {txReceipt
-            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions above.'
+            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions bellow.'
```
Comment (W3stside) [inline src/components/PoolingWidget/AddFunding.tsx:27]:
```suggestion
            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions below.'
```

bellow means to scream, or yell out

Comment (alfetopito) [reply src/components/PoolingWidget/AddFunding.tsx]: Of course I meant that :D

Comment (alfetopito) [reply src/components/PoolingWidget/AddFunding.tsx]: Also:

Comment (W3stside) [reply src/components/PoolingWidget/AddFunding.tsx]: "FRIENDS ONLY" - show the whole picture bro, fake news

**Pull request merged**